ambry-store/src/main/java/com.github.ambry.store/StoreMessageReadSet.java

```diff
@@ -146,6 +148,26 @@ public void close() {
       segment.closeView();
     }
   }
+
+  public void preFetch() {
```

Comment:
doc

Reply:
also tests?

Reply:
doc added. 

It's tested by StoreMessageReadSetTest. Not direct, by covered by its test.